cumulusci.yml

```diff
@@ -352,3 +352,40 @@ orgs:
         enterprise:
             config_file: orgs/enterprise.json
 
+plans:
+    existing_org:
+        slug: install
+        title: Install NPSP
+        tier: primary
+        steps:
+            1:
+                flow: dependencies
+                ui_options:
+                    deploy_pre:
+                        account_record_types:
+                            name: Account Record Types
+                        opportunity_record_types:
+                            name: Opportunity Record Types
+            2:
+                task: install_managed
+            3:
+                task: deploy_post
```

Comment:
@jlantz what user-facing name should we give this step? The current installer has `NPSP Config for Salesforce1` which is pretty ambiguous.

Reply:
NPSP Config for Salesforce Mobile App?

Might want to inquire with the NPSP dev team to get their thoughts though.

Reply:
Ohh.... (lightbulb goes on) ... those components are focused the mobile experience, aren't they. Now I understand the existing name better.